package.json

```
@@ -15,7 +15,7 @@
     "build": "run-s build:*",
     "build:remix": "remix build",
     "build:worker": "remix-pwa build",
-    "dev": "run-p dev:*",
+    "dev": "run-p dev:remix dev:worker",
```

The `dev:*` syntax no longer seems to work in the current yarn version.

---

`run-p` is provided by the `npm-run-all` package, so its syntax support shouldn't change.  I guess what changed is how `yarn` modern parses these scripts.

I can get it working again by quoting the argument.

```suggestion
    "dev": "run-p 'dev:*'",
```

The build script needs updating too.